### PR TITLE
Tipping Readme.

### PR DESCRIPTION
**Pull request by Halabitimoty**
This is the Tipping Readme for Gibwork.

### CODE REVIEW

File: how-it-works/tipping.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
-# Tipping
+# 💡 Tipping System: Acknowledging and Rewarding Your Contributions 🌟
```
Comment (DracocardZ) [inline how-it-works/tipping.md:2]: bug

File: how-it-works/tipping.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
 
-As a open source maintainer, you have the opportunity to reward your community outside of bounties.
```
Comment (DracocardZ) [inline how-it-works/tipping.md:3]: https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github

File: how-it-works/tipping.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
-# Tipping
```
Comment (DracocardZ) [inline how-it-works/tipping.md:1]: tipping=level

File: how-it-works/tipping.md (hunk 2 above)
Comment (DracocardZ) [reply how-it-works/tipping.md:3]: who can i fix this issuse

File: how-it-works/tipping.md (hunk 3 above)
Comment (Halabitimoty) [reply how-it-works/tipping.md:1]: fixed

**Comment by anthonyliriano** [comment]
Thanks for participating @Halabitimoty; we chose another submission. Best of luck in the next bounty.

**Pull request closed without merging**